backend/src/libs/types/service.type.ts

```diff
@@ -5,7 +5,7 @@ type Service<T = unknown> = {
   }>;
   create(payload: unknown): Promise<T>;
   update(): Promise<T>;
-  delete(): Promise<boolean>;
+  delete(payload: unknown): Promise<boolean>;
```

Comment:
```suggestion
  delete(id: number): Promise<boolean>;
```

Reply:
But I need to pass userId too. Isn't it better to have 1 payload instead of 2 params : id and userId?

Reply:
I think you can pass `{ id: number, userId: number }`then